src/client/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import App from './App'
+
+// You can remove this test
```

Comment:
I think you could remove the comment and just call the test 'Example test' and that should be implied.

Reply:
```suggestion
// Example test
```